lib/powerline.sh

```diff
@@ -80,13 +93,13 @@ __process_segment_defaults() {
 			${input_segment[7]:-$separator_disable} \
 		)
 
-		powerline_segments[$segment_index]="${powerline_segment_with_defaults[@]}"
+		powerline_segments[segment_index]="${powerline_segment_with_defaults[@]}"
```

Comment:
Why remove `$` here, and below? A spellcheck recommendation? If so, could you please link 🤔

Reply:
ye, it seems as though when using a value as an array accessor you don't need the variable O.o

https://www.shellcheck.net/wiki/SC2004

It looks as though it's derived from arithmetic rules, however, which is curious

Reply:
Oh interesting, thanks for sharing!